spec/index.bs

```diff
@@ -1409,62 +1409,63 @@ interface GPUBufferUsage {
 };
 </script>
 
-### <dfn method for=GPUDevice>createBuffer(descriptor)</dfn> ### {#GPUDevice-createBuffer}
-
-<div algorithm="GPUDevice.createBuffer">
-  **Arguments:**
-    - {{GPUBufferDescriptor}} |descriptor|
+<dl dfn-type=method dfn-for=GPUDevice>
+    : <dfn>createBuffer(descriptor)</dfn>
+    ::
 
-  **Returns:** {{GPUBuffer}}
+    <div algorithm=GPUDevice.createBuffer>
+        **Called on:** {{GPUDevice}} |this|.
 
-  1. If this call doesn't follow the [$createBuffer Valid Usage$]:
+        **Arguments:**
+        <pre class=argumentdef for="GPUDevice/createBuffer(descriptor)">
+            |descriptor|:
+        </pre>
 
-    1. Retun an error buffer.
+        **Returns:** {{GPUBuffer}}
 
-    Issue(gpuweb/gpuweb#605): Explain that the resulting error buffer can still be mapped at creation.
+        - If any of the following conditions are unsatisfied, return an error buffer and stop.
```

Comment:
Do you have an opinion about the change from numbered lists to bulleted lists? I honestly didn't think about it very hard. On the one hand, I don't think the numbers are all that useful, but on the other hand, they're visually like a different bullet type that clearly says "this is a step" rather than "this is an item in some list that happens to be inside the list of steps".

Reply:
I have no particular opinion aside from "consistency is good". In WebXR we used numbered lists everywhere, and it looks fine, but it's not a critical thing either way in my opinion. I will say that in WebXR we don't have the same style of validation conditions that WebGPU has everywhere. Given that there's not really a order to those it seems like bullets are more appropriate. But if you wanted numbers elsewhere then you're intermixing styles. There's a logic to it, so it wouldn't be problematic, but I don't know if it's worth the trouble.

Reply:
I think it would make sense to intermix the styles, but I also don't think it's worth the trouble. It won't be that hard if we decide someday we want to change it everywhere, so let's leave it for now - I think the bullets are just fine.

Reply:
edit: I think either way is fine, I just saw you changed it to numbers, so don't spend time to change it back :)